third-party/rust/BUCK

```diff
@@ -15495,57 +15525,23 @@ cargo.rust_library(
 
 alias(
     name = "rustls",
-    actual = ":rustls-0.22.4",
+    actual = ":rustls-0.23.19",
     visibility = ["PUBLIC"],
 )
 
 http_archive(
-    name = "rustls-0.22.4.crate",
-    sha256 = "bf4ef73721ac7bcd79b2b315da7779d8fc09718c6b3d2d1b2d94850eb8c18432",
-    strip_prefix = "rustls-0.22.4",
-    urls = ["https://static.crates.io/crates/rustls/0.22.4/download"],
+    name = "rustls-0.23.19.crate",
+    sha256 = "934b404430bb06b3fae2cba809eb45a1ab1aecd64491213d7c3301b88393f8d1",
+    strip_prefix = "rustls-0.23.19",
+    urls = ["https://static.crates.io/crates/rustls/0.23.19/download"],
     visibility = [],
 )
 
 cargo.rust_library(
-    name = "rustls-0.22.4",
-    srcs = [":rustls-0.22.4.crate"],
+    name = "rustls-0.23.19",
+    srcs = [":rustls-0.23.19.crate"],
     crate = "rustls",
-    crate_root = "rustls-0.22.4.crate/src/lib.rs",
-    edition = "2021",
-    features = [
-        "default",
-        "log",
-        "logging",
-        "ring",
-        "tls12",
-    ],
```

Comment:
I'm reasonbly sure we don't want `default-features = false` as `rustls` won't have feature support anymore for `tls12`/`ring`/etc